oedatamodel/latest/v112/datapackage/OEDataModel-normalization-datapackage/OEDataModel-normalization-datapackage.json

```diff
@@ -1,268 +1,259 @@
 {
-    "name": "Oedatamodel - General Energy Modell Datapackage",
-    "title": "OpenEnergyPlatform data format for scenario data",
-    "id": "",
-    "description": "datamodel, metadata and examples provided as datapackage",
-    "language": ["en-GB"],
-    "keywords": ["datamodel", "datapackage", "genral energy dataformat"],
-    "publicationDate": "2020-08-11",
-    "context": {
-        "homepage": "https://openenergy-platform.org/",
-        "documentation": "https://github.com/OpenEnergyPlatform/oedatamodel/blob/develop/README.md",
-        "sourceCode": "https://github.com/OpenEnergyPlatform/oedatamodel/tree/develop/oedatamodel/latest",
-        "contact": "",
-        "grantNo": "",
-        "fundingAgency": "",
-        "fundingAgencyLogo": "",
-        "publisherLogo": ""
-    },
-    "spatial": {
-        "location": "",
-        "extent": "",
-        "resolution": ""
-    },
-    "temporal": {
-        "referenceDate": "",
-        "timeseries": {
-            "start": "",
-            "end": "",
-            "resolution": "",
-            "alignment": "",
-            "aggregationType": ""
-        }
-    },
-    "sources": [
-        {
-            "title": "Open energy datamodel",
-            "description": "oedatamodel for energy model data",
-            "path": "https://github.com/OpenEnergyPlatform/oedatamodel/tree/develop/oedatamodel",
-            "licenses": [
-                {
-                    "name": "CC0-1.0",
-                    "title": "Creative Commons Zero v1.0 Universal",
-                    "path": "https://creativecommons.org/publicdomain/zero/1.0/legalcode",
-                    "instruction": "You are free: To Share, To Create, To Adapt",
-                    "attribution": "© Reiner Lemoine Institut"
-                }
-            ]
-        }
-    ],
-    "licenses": [
+  "name": "Oedatamodel - General Energy Modell Datapackage",
+  "title": "OpenEnergyPlatform data format for scenario data",
+  "id": "",
+  "description": "datamodel, metadata and examples provided as datapackage",
+  "language": ["en-GB"],
+  "subject": [],
+  "keywords": ["datamodel", "datapackage", "general energy dataformat"],
+  "publicationDate": "2020-06-09",
+  "context": {
+    "homepage": "https://openenergy-platform.org/",
+    "documentation": "https://github.com/OpenEnergyPlatform/oedatamodel/blob/develop/README.md",
+    "sourceCode": "https://github.com/OpenEnergyPlatform/oedatamodel/tree/develop/oedatamodel/latest",
+    "contact": "",
+    "grantNo": "",
+    "fundingAgency": "",
+    "fundingAgencyLogo": "",
+    "publisherLogo": ""
+  },
+  "spatial": {
+    "location": "",
+    "extent": "",
+    "resolution": ""
```

Comment:
Same here: Empty keys should be `null` or `todo` in oem151 and not `""`, see line 188-189 in the comments section.